src/components/SearchResults.js

```diff
@@ -1,19 +1,17 @@
 import React from 'react';
 import axios from 'axios';
-import Service from './service.jsx';
-import MapResults from './maps.jsx';
+import Service from './Service';
+import MapResults from './MapResults';
 import { Button, Alert } from 'react-bootstrap';
 import FontAwesome from 'react-fontawesome';
 
 class SearchResults extends React.Component {
   constructor(props) {
     super(props);
     this.resourceId = '35de6bf8-b254-4025-89f5-da9eb6adf9a0';
-    this.toggleShowMap = this.toggleShowMap.bind(this);
     this.state = {results: [], show_map: false};
-    this.fetchResults = this.fetchResults.bind(this);
   }
-  toggleShowMap() {
+  toggleShowMap = () => {
```

Comment:
why do we do this syntax? i've not seen it before.

Reply:
@Br3nda This is an arrow function from ES2015 which is transpiled with babel. It allows to avoid binding this.